src/OrbitGl/GpuTrack.cpp

```diff
@@ -270,10 +270,24 @@ const TextBox* GpuTrack::GetDown(const TextBox* textbox) const {
   }
 }
 void GpuTrack::OnCollapseToggle(TriangleToggle::State state) {
-  Track::OnCollapseToggle(state);
   // When being collapsed, we only show the submission track, and we want those timers also being
-  // collapsed. The marker track does not need to be touched, as it will not be drawn.
+  // collapsed. Further, when expanding the track the user likely wants to get more insights, thus
```

Comment:
Doesn't this reset the collapsed state of the subtracks every time you expand or collapse the parent track? That could be annoying.

Could we do it that the subtracks start expanded even if the parent track starts collapsed, so that when you expand the parent track for the first time the subtracks are expanded, but then their collapsed state is independent of the parent track? (I hope what I'm saying makes sense.)